NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -80,42 +89,31 @@ class EditorViewController: UIViewController {
     private func setupUI() {
         view.backgroundColor = UIColor(hex: "#252525")
         
-        setupCustomBackButton()
-        
         titleTextView.addDoneButtonToolbar()
         bodyTextView.addDoneButtonToolbar(onDone: (target: self, action: #selector(customDoneAction)))
         
         view.addSubview(titleTextView)
         view.addSubview(bodyTextView)
-        
-        titleTextView.translatesAutoresizingMaskIntoConstraints = false
-        bodyTextView.translatesAutoresizingMaskIntoConstraints = false
-        
-        NSLayoutConstraint.activate([
-            titleTextView.topAnchor.constraint(equalTo: customBackButton.bottomAnchor, constant: 20),
-            titleTextView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
-            titleTextView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
-            
-            bodyTextView.topAnchor.constraint(equalTo: titleTextView.bottomAnchor, constant: 10),
-            bodyTextView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
-            bodyTextView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
-            bodyTextView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 20)
-        ])
-    }
-
-    private func setupCustomBackButton() {
-        customBackButton.addAction(UIAction(handler: { [weak self] _ in
-            self?.backButtonTapped()
-        }), for: .touchUpInside)
         view.addSubview(customBackButton)
         
-        customBackButton.translatesAutoresizingMaskIntoConstraints = false
-        
+        setupHierarchy()
+    }
+    
+    private func setupHierarchy() {
```

Comment:
Same here this func should have name related to constants, setuphierarchy can be used for naming where we add subviews